### DISCUSSION TITLE
Evernote Migration

### DISCUSSION THREAD

**Discussion by mahert45** [mealie-recipes/mealie#4578] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

My recipes are currently in Evernote, which is changing up their prices.  Being able to take the ENEX file from Evernote and import would be amazing

### Please provide a concise description of the feature that would resolve your issue.

My recipes are currently in Evernote, which is changing up their prices.  Being able to take the ENEX file from Evernote and import would be amazing

### Please consider and list out some caveats or tradeoffs made in your design decision

If it could work, that's all I'm looking for

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
Evernote files are unstructured, they're basically just text files. You can try importing them via OpenAI/GPT. We also have an [open PR](https://github.com/mealie-recipes/mealie/pull/4561) for importing text via OpenAI directly in Mealie.
